src/gui/clientproxy.cpp

```diff
@@ -53,7 +53,7 @@ bool ClientProxy::isUsingSystemDefault()
         return cfg.proxyType() == QNetworkProxy::DefaultProxy;
     }
 
-    return false;
+    return true;
```

Comment:
What is this change doing? By the commit message, I would just expect strings to change, however, here is a behavioral change.

Reply:
I didn't change this, this is due to a rebase conflict, I will rebase and amend-push